designer/server/src/main/resources/defaultDesignerConfig.conf

```diff
@@ -21,6 +21,10 @@ db {
   minimumIdle: 1
   #has to be lower <= maxConnections
   numThreads: 5
+  properties: {
+    # PostgreSQL specific
+    "ApplicationName": "Nussknacker"
```

Comment:
What is it? Why it is in defaultDesignerConfig?

Reply:
To niczego nie psuje. Jak opisano w komentarzu powyżej: jest to bardziej specyficzne dla postgresql. Aby uniknąć nadpisywania konfiguracji DB, gdy uruchamiamy NU z PostgreSQL jako persistence. Ta właściwość jest używana do zadeklarowania / nazwania połączenia wykonanego przez Nussknacker do DB.